app/src/main/java/ai/elimu/analytics/entity/StoryBookLearningEvent.java

```diff
@@ -10,6 +10,7 @@ public class StoryBookLearningEvent extends LearningEvent {
 
     @NonNull
     private Long storyBookId;
+    private String storyBookTitle;
```

Comment:
Add empty line between properties, just to make it extra clear that the `storyBookTitle` is not also `@NonNull`.

```suggestion

    private String storyBookTitle;
```

Reply:
Makes sense! Did the changes